.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+on:
+  push:
+    tags:
+      - '**'
```

Comment:
For other repos we've used the `release` event. But this repo doesn't currently use releases.

Instead, Lerna pushes some tags. Maybe I'll look at changing this in the future, though 🤔